docs/content/usage/front-matter.mdx

```diff
@@ -38,13 +38,12 @@ status: Stable
 
 Doctocat comes with a few predefined statuses that have colors associated with them:
 
-| Status         | Output                                |
-| -------------- | ------------------------------------- |
-| `Stable`       | <StatusLabel status="Stable" />       |
-| `New`          | <StatusLabel status="New" />          |
-| `Experimental` | <StatusLabel status="Experimental" /> |
-| `Review`       | <StatusLabel status="Review" />       |
-| `Deprecated`   | <StatusLabel status="Deprecated" />   |
+| Status         | Output                                | Meaning                                                                |
+| -------------- | ------------------------------------- |------------------------------------------------------------------------|
+| `Alpha`        | <StatusLabel status="Alpha" />        | Experimental. Usage requires review from a Design Systems team member. |
```

Comment:
Do we want to try to use language that may make sense outside of GitHub? "Design Systems", I assume, doesn't mean anything to anyone else... 

I'm not sure what the right language is those. We could just drop the whole "review" lanuage?

```suggestion
| `Alpha`        | <StatusLabel status="Alpha" />        | Experimental. Breaking changes are likely. |
```